### PR TITLE
Feature/pr 003 bonus to 25per

### PR DESCRIPTION
**Pull request by sachinmagadum**

### CODE REVIEW

File: BonusCal.py [hunk 1]
```diff
@@ -2,12 +2,11 @@
...
-bonus=(0.15*sal)
+# bonus modified with 25%
+# modified date 11dec at 11.00am
+
+bonus=(0.25*sal)
 if exp>=2 and sal>=25000:
     print(f"Your bonus amount is Rs.{bonus}")
 else:
     print(f"Sorry, you are not eligible for bonus")
-
```
Comment (Pramodyadav07) [inline BonusCal.py:10]: why these lines are removed?

Comment (sachinmagadum) [reply BonusCal.py:10]: These lines are added by me in last PR. and these line have no use.

**Review by Pramodyadav07 — APPROVED** [review]
looks good procced

**Review by VAIBHAV-00 — APPROVED** [review]
look good

**Pull request merged**